travel.py

```diff
@@ -35,14 +37,20 @@
                     help='Password of user with access to bucket')
 args = parser.parse_args()
 
-## CB Server
-
 if not args.cluster:
   raise ConnectionError("No value for CB_HOST set!")
 
 CONNSTR = "{scheme}://{cluster}{connectargs}".format(**vars(args))
 authenticator = PasswordAuthenticator(args.user, args.password)
 
+# get a reference to our cluster
+options = ClusterOptions(authenticator)
+
+# Sets a pre-configured profile called "wan_development" to help avoid latency issues
+# when accessing Capella from a different Wide Area Network
+# or Availability Zone(e.g. your laptop).
+options.apply_profile('wan_development')
```

Comment:
will this cause problems for non-capella?

Reply:
actually, I just made it a conditional if this is a capella call, then we use it. Doesn't seem to make sense to use for couchbase server since its using localhost